src/models/userDao.js

```diff
@@ -0,0 +1,40 @@
+const { appDataSource } = require("./appDataSource");
+
+const getUserInfo = async (socialId, socialTypeId) => {
+  const userInfo = await appDataSource.query(
+    `SELECT
+      id
+    FROM
+      users
+    WHERE
+      social_id = ? AND social_type_id = ?`,
+    [socialId, socialTypeId]
+  );
+
+  return userInfo;
+};
+
+const createUser = async (socialId, email, nickname, socialTypeId) => {
+  const newUser = await appDataSource.query(
+    `INSERT INTO users (
+      social_id,
+      email,
+      nickname,
+      social_type_id
+    ) 
+    VALUES (
+      ?, 
+      ?, 
+      ?, 
+      ?
+    )`,
+    [socialId, email, nickname, socialTypeId]
+  );
+
+  return newUser;
+};
```

Comment:
```suggestion
const createUser = async (socialId, email, nickname, socialTypeId) => {
  return appDataSource.query(
    `INSERT INTO users (
      social_id,
      email,
      nickname,
      social_type_id
    ) 
    VALUES (
      ?, 
      ?, 
      ?, 
      ?
    )`,
    [socialId, email, nickname, socialTypeId]
  );
};
```
변수 선언과 await 없이 곧바로 리턴해주는것이 더 깔끔 할 것 같아요.